### PR TITLE
Openapi/tlds sip48

### PR DESCRIPTION
**Pull request by ohager**
- [x] Add `getAliasesByName`
- [x] Add SetTLD
- [x] Extend Alias response by `tld` and `tldName`
- [x] Extend Alias Functions like `sellAlias`, `buyAlias`, etc by `tld`

### CODE REVIEW

**Comment by frankTheTank72** [comment]
getAliases
This also has a new parameter - aliasName

**Comment by frankTheTank72** [comment]
buyAlias
This also has a new parameter - aliasName - tld
**tld** should be the name

File: openapi/signum-api.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.3",
   "info": {
     "title": "Signum Node API",
-    "version": "3.5.1",
+    "version": "3.6.1",
```
Comment (frankTheTank72) [inline openapi/signum-api.json:5]: i think 3.7.0

File: openapi/paths/alias/setTLD.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+  "post": {
+    "summary": "Set Custom Alias TLD",
+    "description": "Create a [Top Level Domain (TLD)](https://github.com/signum-network/SIPs/blob/master/SIP/sip-48.md) for aliases. A TLD is like a namespace for Aliases, such it is possible to have same Aliases in different TLDs. Note, that acquisition of a custom TLD charges 100K SIGNA",
```
Comment (frankTheTank72) [inline openapi/paths/alias/setTLD.json:4]: Mabye saying that this is to pay as amount (100k) - or can we fix the value of the amount field with 100k ?

Comment (ohager) [reply openapi/paths/alias/setTLD.json]: I removed it

**Review by frankTheTank72 — APPROVED** [review]

**Pull request merged**